templates/resources/page_rule.md

```diff
@@ -80,6 +80,13 @@ Action blocks support the following:
 - `true_client_ip_header` - (Optional) Whether this action is `"on"` or `"off"`.
 - `waf` - (Optional) Whether this action is `"on"` or `"off"`.
 
+Cache Level actions support the following levels:
+ - `"bypass"` - Bypass - Cloudflare does not cache.
+ - `"basic"` - No Query String - Delivers resources from cache when there is no query string. Example URL: `example.com/pic.jpg`
+ - `"simplified"` - Ignore Query String - Delivers the same resource to everyone independent of the query string. Example URL: `example.com/pic.jpg?ignore=this-query-string`
+ - `"aggressive"` - Standard - Caches all static content that has a query string. Example URL: `example.com/pic.jpg?with=query`
+ - `"cache_everything"` - Cache Everything -  Treats all content as static and caches all file types beyond the [Cloudflare default cached content](https://developers.cloudflare.com/cache/about/default-cache-behavior#default-cached-file-extensions).  Respects cache headers from the origin web server unless Edge Cache TTL is also set in the Page Rule. When combined with an Edge Cache TTL > 0, Cache Everything removes cookies from the origin web server response. 
+
```

Comment:
```suggestion
```

Reply:
The goal of my PR is ultimately to map the `cache_level` options — which use the API values — to the Dashboard values. The referenced documentation links do not fully provide that mapping.

If including the information about what each value does is considered too much and duplicating information, perhaps a middle ground is to only provide the map. Something more simple like

```
- `"basic"` (No Query String)
- `"simplified"` (Ignore Query String)
```

Reply:
the terraform documentation shouldn't be the source of truth for this type of documentation. instead, we should lean on the developer docs or API docs which are managed by our content team and have a much better strategy for this type of thing. it also means that folks outside of Terraform benefit from the improvements.

Reply:
i'm open to linking to https://developers.cloudflare.com/cache/how-to/set-caching-levels/ instead if you prefer?